Source/Management.GraphQL/EventHandlers/EventHandler.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.Runtime.Management.GraphQL.EventHandlers
+{
+    public class EventHandler
+    {
+        public Guid Id { get; set; }
+        public Guid Scope { get; set; }
+        public int FilterPosition { get; set; }
+        public int EventProcessorPosition { get; set; }
+        public int TailEventLogSequenceNumber { get; set; }
```

Comment:
See above comments of tail-event-sequence-number. It is fine to have an int here for this and return -1 when nothing is committed.

I would maybe call the `TailEventLogSequenceNumber` the `LastCommittedEventSequenceNumber`. And since it seems like you want to report how far along the event handler has gotten - I think you want to have `LastFilteredEventStreamPosition` as well.